models/staging/stg_claimable_balances.yml

```diff
@@ -0,0 +1,121 @@
+version: 2
+
+models:
+  - name: stg_claimable_balances
+    tests:
+      - dbt_utils.recency:
+          datepart: day
+          field: cast(closed_at as timestamp)
+          interval: 2
+          meta:
+            description: "Monitors the freshness of your table over time, as the expected time between data updates."
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - balance_id
+            - ledger_entry_change
+          meta:
+            description: "Tests the uniqueness combination of: balance_id and ledger_entry_change."
+    columns:
+      - name: balance_id
+        description: '{{ doc("balance_id") }}'
+        tests:
+          - not_null
+
+      - name: claimants
+        description: '{{ doc("claimants") }}'
+
+      - name: claimants.destination
+        description: '{{ doc("claimants_destination") }}'
+
+      - name: claimants.predicate
+        description: '{{ doc("claimants_predicate") }}'
+
+      - name: claimants.predicate.unconditional
+        description: '{{ doc("claimants_predicate_unconditional") }}'
+
+      - name: claimants.predicate.abs_before
+        description: '{{ doc("claimants_predicate_abs_before") }}'
+
+      - name: claimants.predicate.rel_before
+        description: '{{ doc("claimants_predicate_rel_before") }}'
+
+      - name: claimants.predicate.abs_before_epoch
+        description: '{{ doc("claimants_predicate_abs_before_epoch") }}'
+
+      - name: asset_type
+        description: '{{ doc("asset_type") }}'
+        tests:
+          - not_null
+
+      - name: asset_code
+        description: '{{ doc("asset_code") }}'
+        tests:
+          - not_null
+
+      - name: asset_issuer
+        description: '{{ doc("asset_issuer") }}'
+        tests:
+          - not_null
```

Comment:
These fields are blank (null) for native `XLM` balances. Dunno how effective these tests are

Reply:
Resolved in https://github.com/stellar/stellar-dbt-public/pull/100/commits/e41f9a26bc1044392dc3c92eba23a10f8d995fad